helper/helper/src/main/java/com/linkedin/avroutil1/compatibility/AvroCompatibilityHelper.java

```diff
@@ -602,4 +602,28 @@ public static String getSchemaFullName(Schema schema) {
       return schema.getName();
     }
   }
+
+  /**
+   * Create a new schema field with provided arguments. This adds compatibility layer for {@link Schema.Field} before
+   * avro 1.9, which takes JsonNode as default value in constructor instead of Object.
+   * @param name the name of the field
+   * @param schema the schema of the field
+   * @param doc the doc of the field
+   * @param defaultValue the default value of the field. For avro 1.4-1.8, user should pass in a JsonNode. For avro 1.9
```

Comment:
you should clarify which methods on avro class Schema() or elsewhere these defauult valuues should come from